File: Java/Tram.java
import java.util.Map;
import java.util.HashMap;
import java.time.DayOfWeek;

public class Tram extends Transportation {
    public Tram() {
        super(new HashMap<DayOfWeek, Integer>() {{
            put(DayOfWeek.MONDAY, 25);
            put(DayOfWeek.TUESDAY, 25);
            put(DayOfWeek.WEDNESDAY, 25);
            put(DayOfWeek.THURSDAY, 25);
            put(DayOfWeek.FRIDAY, 25);
            put(DayOfWeek.SATURDAY, 0);
            put(DayOfWeek.SUNDAY, 0);
        }});
    }
}

File: C#/KnowItTest/Transportations/Tram.cs
public class Tram : Transportation
{
    public Tram() : base(new Dictionary<DayOfWeek, int>
      {
          { DayOfWeek.Monday, 25 },
          { DayOfWeek.Tuesday, 25 },
          { DayOfWeek.Wednesday, 25 },
          { DayOfWeek.Thursday, 25 },
          { DayOfWeek.Friday, 25 },
          { DayOfWeek.Saturday, 0 },
          { DayOfWeek.Sunday, 0 }
      })
    {
        // Constructor body 
    }
}
